jobs/upload-brew.Jenkinsfile

```diff
@@ -0,0 +1,128 @@
+import org.yaml.snakeyaml.Yaml;
+
+def pipeutils, pipecfg, libcloud, uploading, jenkins_agent_image_tag
+def src_config_url, src_config_ref, brew_principle, brew_profile
+node {
+    checkout scm
+    pipeutils = load("utils.groovy")
+    pipecfg = pipeutils.load_pipecfg()
+    libcloud = load("libcloud.groovy")
+
+    brew_principle =  pipecfg['brew']['principle']
+    brew_profile =  pipecfg['brew']['profile']
+}
+
+arches = [ 'aarch64', 'ppc64le', 'x86_64', 's390x']
```

Comment:
we have a `get_supported_additional_arches()` function for this, but we should probably just default to pushing all arches that the given version was built against.